DigitalRain/Raindrops/StandardRaindrop.cs

```diff
@@ -58,30 +62,33 @@ public char GetSymbolFromPool()
             return SymbolPool[index];
         }
 
-        // IRaindrop
+        float lerpScaleFactor = 350;
+        float lerpAmount = 400;
```

Comment:
OK BEFORE YOU FREAK OUT!

I am working on getting this rolled into the config class. But I am currently modifying that class in my `global_config_access` branch. So I wanted to merge this first so the changes are more visible.

Reply:
Hmmm again I don't think we need both these values.
It looks like the lerp method takes a percentage (between 0 and 1).  So I imagine you just want to map that to a timeframe much like we did with lifespan to alpha.  So if I want this effect to last for 3 seconds, at t=0 I want a lerpAmount of 0.0 and at t >= 3 I want a lerpAmount of 1.0.

Reply:
Feel free to change this! This was the best I could do with my smooth brain